versions.lock

```diff
@@ -64,125 +64,83 @@ jakarta.activation:jakarta.activation-api:1.2.2 (2 constraints: ba28fcbd)
+io.undertow:undertow-servlet:2.2.18.Final (1 constraints: 57074361)
+junit:junit:4.13.2 (2 constraints: 43140975)
 net.bytebuddy:byte-buddy:1.12.10 (2 constraints: e9163f66)
 net.bytebuddy:byte-buddy-agent:1.12.10 (1 constraints: 6f0ba8e9)
 net.jcip:jcip-annotations:1.0 (1 constraints: 560ff165)
-net.sourceforge.argparse4j:argparse4j:0.8.1 (1 constraints: 430d3a1f)
 org.apache-extras.beanshell:bsh:2.0b6 (1 constraints: ac07626b)
-org.apache.commons:commons-lang3:3.12.0 (4 constraints: 2d31298e)
-org.apache.commons:commons-text:1.8 (1 constraints: b4102e9b)
-org.assertj:assertj-core:3.23.1 (3 constraints: 5e28a371)
+org.apache.commons:commons-lang3:3.12.0 (1 constraints: 38053b3b)
+org.apiguardian:apiguardian-api:1.1.2 (5 constraints: 105480ac)
+org.assertj:assertj-core:3.23.1 (2 constraints: 9819aede)
 org.conscrypt:conscrypt-openjdk-uber:1.4.1 (1 constraints: 0805fd35)
-org.eclipse.jetty:jetty-continuation:9.4.24.v20191120 (2 constraints: 7021f842)
-org.eclipse.jetty:jetty-http:9.4.24.v20191120 (3 constraints: 03312e51)
-org.eclipse.jetty:jetty-io:9.4.24.v20191120 (3 constraints: ea2fc7ae)
-org.eclipse.jetty:jetty-security:9.4.24.v20191120 (1 constraints: 64103ae2)
-org.eclipse.jetty:jetty-server:9.4.24.v20191120 (5 constraints: a4551466)
-org.eclipse.jetty:jetty-servlet:9.4.24.v20191120 (2 constraints: 152094cd)
-org.eclipse.jetty:jetty-servlets:9.4.24.v20191120 (1 constraints: 381082d7)
-org.eclipse.jetty:jetty-util:9.4.24.v20191120 (5 constraints: c84d7896)
-org.eclipse.jetty:jetty-webapp:9.4.24.v20191120 (1 constraints: 9a109aeb)
-org.eclipse.jetty:jetty-xml:9.4.24.v20191120 (1 constraints: de0f18cb)
-org.eclipse.jetty.toolchain.setuid:jetty-setuid-java:1.0.3 (1 constraints: 3e0d291f)
-org.glassfish.jersey.containers:jersey-container-servlet:2.34 (1 constraints: 610e824e)
-org.glassfish.jersey.containers:jersey-container-servlet-core:2.34 (2 constraints: 9e317578)
+org.glassfish.jersey.containers:jersey-container-servlet-core:2.34 (1 constraints: dd04fd30)
 org.glassfish.jersey.ext:jersey-entity-filtering:2.34 (1 constraints: 91156ebd)
-org.glassfish.jersey.ext:jersey-metainf-services:2.34 (1 constraints: 610e824e)
-org.glassfish.jersey.media:jersey-media-jaxb:2.34 (1 constraints: e01957ca)
 org.glassfish.jersey.media:jersey-media-json-jackson:2.34 (1 constraints: dd04fd30)
-org.glassfish.jersey.test-framework:jersey-test-framework-core:2.34 (1 constraints: 4b23d8ba)
-org.glassfish.jersey.test-framework.providers:jersey-test-framework-provider-inmemory:2.34 (1 constraints: cd0ef25f)
 org.hamcrest:hamcrest:2.1 (2 constraints: 8512932d)
 org.hamcrest:hamcrest-all:1.3 (1 constraints: a8041d2c)
 org.hamcrest:hamcrest-core:2.1 (2 constraints: 88126e10)
 org.hamcrest:hamcrest-library:2.1 (1 constraints: 1507415c)
+org.jboss.spec.javax.annotation:jboss-annotations-api_1.3_spec:2.0.1.Final (1 constraints: 150fc165)
+org.jboss.spec.javax.servlet:jboss-servlet-api_4.0_spec:2.0.0.Final (1 constraints: 140fba65)
```

Comment:
nit: let's exclude these in favor of the standard servlet/annotation-apis. I'd expect classpath uniqueness checks to fail here, I'm not sure why they aren't...

Reply:
I have stanzas in my build.gradle which are doing the module replacements...